RHEL/6/input/xccdf/system/software/integrity.xml

```diff
@@ -266,4 +267,77 @@ computer viruses, as well as to limit their spread to other systems.
 
 </Group>
 
+<Group id="fips">
+<title>Federal Information Processing Standard (FIPS)</title>
+<description>
+The Federal Information Processing Standard (FIPS) is a computer security standard which
+is developed by the U.S. Government and industry working groups to validate the quality
+of cryptographic modules. The FIPS standard provides four security levels to ensure
+adequate coverage of different industries, implementation of cryptographic modules, and
+organizational sizes and requirements.
+<br /><br />
+FIPS 140-2 is the current standard for validating that mechanisms used to access cryptographic modules
+utilize authentication that meets industry and government requirements. For government systems, this allows
+Security Levels 1, 2, 3, or 4 for use on Red Hat Enterprise Linux.
+<br /><br />
+See <b>http://csrc.nist.gov/publications/PubsFIPS.html</b> for more information.
+</description>
+
+<Rule id="package_dracut-fips_installed" severity="medium">
+<title>Install the dracut-fips Package</title>
+<description>
+To enable FIPS, the system requires that the <tt>dracut-fips</tt>
+package be installed. To install the <tt>dracut-fips</tt> package:
```

Comment:
Just nitpick, "To enable FIPS, the system requires that the <tt>dracut-fips</tt> package to be installed"?
Like s/package be/package to be/ ?